app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -33,12 +33,25 @@ def update
       end
     end
 
+    def self.opted_in_to_remote(idv_session:)
+      if IdentityConfig.store.in_person_proofing_opt_in_enabled &&
+         IdentityConfig.store.in_person_proofing_enabled &&
+         idv_session.service_provider&.in_person_proofing_enabled
+        idv_session.skip_doc_auth == false
+      else
+        idv_session.skip_doc_auth.nil? || idv_session.skip_doc_auth == false
+      end
+    end
+
     def self.step_info
       Idv::StepInfo.new(
         key: :hybrid_handoff,
         controller: self,
         next_steps: [:link_sent, :document_capture],
-        preconditions: ->(idv_session:, user:) { idv_session.idv_consent_given },
+        preconditions: ->(idv_session:, user:) {
```

Comment:
I am not sure I like opted_in_to_remove. It makes me think if you don't opt in you will not be able to see this page. I can't think of a better to suggest tho. Let me come back to this.